### DISCUSSION TITLE
Cavum Tenebrae?

### DISCUSSION THREAD

**Discussion by JerichoJack** [Povstalec/StargateJourney#58] [Q&A] [answered]
Currently running newest version of ATM9, what do i need to do to add Cavum Tenebrae Dimension?

**Comment by Povstalec** [comment]
You'd need to update the mod, since to my knowledge, ATM9 is still on sgjourney 0.6.8

**Reply by JerichoJack** [reply]
What version would be best to use? Latest, or specific version?

**Reply by Povstalec** [reply, marked as answer]
Probably the latest one

**Reply by JerichoJack** [reply]
so, anything above v 1.20.1-0.6.8 breaks ATM9... reading up in the documentation, could the Cavum Tenebrae Dimension be added to ATM9 running sgjourney 0.6.8 via a datapack???

**Reply by Povstalec** [reply]
Hmmm, your pack breaks most likely because of the Common Stargates Datapack, the old version of which is incompatible with newer versions of Stargate Journey, so you'll also need to update that one

**Comment by JerichoJack** [comment]
> Hmmm, your pack breaks most likely because of the Common Stargates Datapack, the old version of which is incompatible with newer versions of Stargate Journey, so you'll also need to update that one

I'll give that a shot.... about to brew my own modpack at this point lol, but i dont want to make 300+ manual additions...
